ykrt/src/compile/mod.rs

```diff
@@ -25,6 +28,15 @@ pub trait Compiler: Send + Sync {
 }
 
 pub fn default_compiler() -> Result<Arc<dyn Compiler>, Box<dyn Error>> {
+    #[cfg(jitc_yk)]
+    // Transitionary env var to turn on the new code generator.
+    //
+    // This will be removed once the transition away from LLVM as a trace compiler is complete.
+    if let Ok(v) = env::var("YKD_NEW_CODEGEN") {
```

Comment:
`if let Ok("1") = env::var(...)` should work I think?

Reply:
Yeah. Dunno why I did that :laughing:

Reply:
Dose `if let Ok("1") = env::var("YKD_NEW_CODEGEN").map(|x| x.as_str())` work?

Reply:
OK, sorry for leading you up the garden path!